exercises/practice/bottle-song/.meta/example.go

```diff
@@ -0,0 +1,61 @@
+package bottlesong
+
+import (
+	"fmt"
+	"strings"
+)
+
+func Recite(startBottles, takeDown int) []string {
+	verses := []string{}
+	for i := startBottles; i > startBottles-takeDown; i -= 1 {
+		verses = append(verses, verse(i)...)
+
+		if i > startBottles-takeDown+1 {
+			verses = append(verses, "")
+		}
+	}
+	return verses
+}
+
+var numberToWord = map[int]string{
+	1:  "one",
+	2:  "two",
+	3:  "three",
+	4:  "four",
+	5:  "five",
+	6:  "six",
+	7:  "seven",
+	8:  "eight",
+	9:  "nine",
+	10: "ten",
+}
+
+func verse(n int) []string {
+	switch {
+	case n == 1:
+		return []string{
+			"One green bottle hanging on the wall,",
+			"One green bottle hanging on the wall,",
+			"And if one green bottle should accidentally fall,",
+			"There'll be no green bottles hanging on the wall.",
+		}
+	case n == 2:
+		return []string{
+			"Two green bottles hanging on the wall,",
+			"Two green bottles hanging on the wall,",
+			"And if one green bottle should accidentally fall,",
+			"There'll be one green bottle hanging on the wall.",
+		}
+	default:
+		return []string{
+			fmt.Sprintf("%s green bottles hanging on the wall,", toTitleCase(numberToWord[n])),
+			fmt.Sprintf("%s green bottles hanging on the wall,", toTitleCase(numberToWord[n])),
+			"And if one green bottle should accidentally fall,",
+			fmt.Sprintf("There'll be %s green bottles hanging on the wall.", numberToWord[n-1]),
+		}
+	}
+}
+
+func toTitleCase(s string) string {
+	return strings.ToUpper(s[0:1]) + strings.ToLower(s[1:])
+}
```

Comment:
I wasn't able to use `strings.Title` as it has been deprecated in Go 1.18 ([Build for reference](https://github.com/exercism/go/actions/runs/3250339595/jobs/5333890548))

Is there a better way to do this? The build suggests using `golang.org/x/text/cases` instead but that would involve installing an external dependency

Reply:
Temporarily, feel free to copy the `strings.Title` and `strings.isSeparator` function directly: https://github.com/exercism/go/blob/43950ad7c2f1fc054f7487943f6f47a1228ac8af/exercises/practice/accumulate/accumulate_test.go#L10-L55

In the meantime, we are adding support to external packages to the track, and `golang.org/x/text` is one of the dependencies we'll support right out of the bat: https://github.com/exercism/go-test-runner/pull/84

When we have this, we'll replace all the copies of `strings.Title` we have.

Reply:
Thanks! I've added that to `bottle_song_test.go` and updated the example solution